CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ CHANGELOG
 
 - Breaking change for Policy which is in Public Preview: Change `pulumi policy apply` to `pulumi policy enable`, and allow users to specify the Policy Group.
 
+- Add `pulumi policy ls packs` and `pulumi policy ls groups` to list Policy related resources.
```

Comment:
Was there a reason we ended up with:

```
pulumi policy ls packs
pulumi policy ls groups
```

Rather than (per #3589):

```
pulumi policy pack ls
pulumi policy group ls
```

?

Reply:
Yes yes I was being lazy since I didnt want to create the 2 commands for packs / groups and the two separate ones for ls. But alas I shall do it if we think its the right call

Reply:
Thinking about this some more, I'm not sure which variant would be better, TBH.

However, right now we have:

```
pulumi policy publish
pulumi policy rm
pulumi policy enable
pulumi policy disable
```

Which are all about interacting with policy _packs_.

I wonder if we should have the following only for listing policy _packs_:

```
pulumi policy ls
```

And then for listing policy _groups_ have:

```
pulumi policy group ls
```

Then we could add potential future commands related to managing policy _groups_ under `pulumi policy group xxx`, e.g. `pulumi policy group rm`.

Reply:
hmm yeah i think i like `pulumi policy ls` being for a policy pack but I am not sure about the `pulumi policy group ls` 

what if pulumi policy ls output both the groups and packs? Like all policy related resources. Or if it was `pulumi policy-group ls` 🤔

Reply:
Okay going w `pulumi policy ls` and `pulumi policy groups ls`